resources/assets/tconstruct/lang/zh_cn.lang

```diff
@@ -3,40 +3,40 @@
 ################
 
 tile.tconstruct.tooltables.craftingstation.name=合成站
-tile.tconstruct.tooltables.stenciltable.name=模具加工台
-tile.tconstruct.tooltables.partbuilder.name=部件加工台
-tile.tconstruct.tooltables.toolstation.name=工具装配台
+tile.tconstruct.tooltables.stenciltable.name=刻模桌
+tile.tconstruct.tooltables.partbuilder.name=塑件台
+tile.tconstruct.tooltables.toolstation.name=组器站
 tile.tconstruct.tooltables.patternchest.name=模具箱
-tile.tconstruct.tooltables.patternchest.tooltip=破坏可以保留其中的模具
+tile.tconstruct.tooltables.patternchest.tooltip=可存储模具或铸模
 tile.tconstruct.tooltables.partchest.name=部件箱
-tile.tconstruct.tooltables.partchest.tooltip=可以容纳大量的部件
-tile.tconstruct.toolforge.name=工具装配台
-
-tile.tconstruct.soil.grout.name=水泥
-tile.tconstruct.soil.grout.tooltip=我就是渣渣！
-tile.tconstruct.soil.slimy_mud_green.name=绿色黏滑泥泞
-tile.tconstruct.soil.slimy_mud_green.tooltip=在上面行走会变得缓慢，但是玩家可以跳的更高
-tile.tconstruct.soil.slimy_mud_blue.name=蓝色黏滑泥泞
+tile.tconstruct.tooltables.partchest.tooltip=能容纳大量部件
+tile.tconstruct.toolforge.name=铸器砧
+
+tile.tconstruct.soil.grout.name=砖泥
+tile.tconstruct.soil.grout.tooltip=I am Grout!
+tile.tconstruct.soil.slimy_mud_green.name=绿色粘液泥
+tile.tconstruct.soil.slimy_mud_green.tooltip=减慢行走速度，提升玩家跳跃高度
+tile.tconstruct.soil.slimy_mud_blue.name=蓝色粘液泥
 tile.tconstruct.soil.slimy_mud_blue.tooltip=tile.tconstruct.soil.slimy_mud_green.tooltip
-tile.tconstruct.soil.slimy_mud_magma.name=熔岩黏滑泥泞
+tile.tconstruct.soil.slimy_mud_magma.name=岩浆粘液泥
 tile.tconstruct.soil.slimy_mud_magma.tooltip=tile.tconstruct.soil.slimy_mud_green.tooltip
 tile.tconstruct.soil.graveyard.name=黄泉土
-tile.tconstruct.soil.graveyard.tooltip=治愈亡灵类生物
+tile.tconstruct.soil.graveyard.tooltip=治疗亡灵生物
 tile.tconstruct.soil.consecrated.name=神圣土
-tile.tconstruct.soil.consecrated.tooltip=伤害亡灵类生物
+tile.tconstruct.soil.consecrated.tooltip=伤害亡灵生物
 
-tile.tconstruct.deco_ground.mudbrick.name=泥砖
+tile.tconstruct.deco_ground.mudbrick.name=泥砖块
 tile.tconstruct.deco_ground_slab.mudbrick.name=泥砖台阶
 tile.tconstruct.mudbrick_stairs.name=泥砖楼梯
 
 tile.tconstruct.ore.cobalt.name=钴矿石
 tile.tconstruct.ore.ardite.name=阿迪特矿石
 
-tile.tconstruct.metal.tooltip=可以作为信标基座使用
+tile.tconstruct.metal.tooltip=可用于建造信标金字塔
 tile.tconstruct.metal.cobalt.name=钴块
 tile.tconstruct.metal.ardite.name=阿迪特块
 tile.tconstruct.metal.manyullyn.name=玛玉灵块
-tile.tconstruct.metal.knightslime.name=骑士史莱姆块
+tile.tconstruct.metal.knightslime.name=骑士粘石块
```

Comment:
原文有介绍，一句话，是针对这种材料的。可以再@一下作者。

Reply:
P.S.和玛玉灵不同，做这个的作者应该活着。

Reply:
对的，这我是知道的，但是万一和格鲁特一样就麻烦了

Reply:
@SlimeKnights Sorry for this disturbance, I really want to know the organization name has any relation with the [SlimeKnights](https://dragonquest.fandom.com/wiki/Slime_knight)? or it actually has a relation with any other things? or just name it by fits and starts?

Reply:
Yes and no. The metal is a pun on the organization name, but is also in reference to the idea of a "knight slime", who is a slime wearing armor/wielding a sword. A literal translation is probably your best bet. Its unrelated to dragon quest for the record, the slime itself would be the knight, rather than having a knight rider.

Reply:
ok, thanks for reply.

Reply:
> ok, thanks for reply.

那还是回滚骑士史莱姆吧，这个材料实际上是生物性质的。